docker/start.sh

```diff
@@ -40,4 +37,4 @@ then
 fi
 
 chown -R git:git /data .
-exec su git -c "./gogs web"
+exec supervisord -c /app/gogs/docker/supervisord.conf
```

Comment:
Cause supervisor runs sshd too.

In docker you must run only one master process. In this case the master process is `supervisord` and it have `sshd` and `gogs` as child processes.

Ah, and this must fix #1499. :wink: